server/plugin.go

```diff
@@ -415,7 +414,12 @@ func (p *Plugin) GetSiteURL() string {
 }
 
 func (p *Plugin) CreateFullURLPath(extensionPath string) string {
-	return path.Join(p.GetSiteURL(), p.GetPluginURLPath(), extensionPath)
+	baseURL, err := url.Parse(p.GetSiteURL())
+	if err != nil {
+		return ""
```

Comment:
are u checking this empty string anywhere? 
Have you also tested this thoroughly

Reply:
Yes, tested